tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+envlist = py36-django{22,32}
```

Comment:
Can we also add `py38-django{22,32} to this list? edX has already migrated all their services to Python 3.8, so we should make sure this xblock works on 3.8 as well.